README.md

```diff
@@ -128,12 +128,26 @@ Then copy over `packages.json` into the directory:
 
 If this builds correctly, you can then push up this branch and make a pull request. Travis will verify your package builds correctly, and then we will try to merge your pull request. Your package will then be available in the next tagged package set.
 
+### Formatting the package set
+
+When creating your pull request, make sure to run the `format` command to pretty-print the packages file. This helps us avoid problems in the future with git diffs and so on.
+
 ### Update a package in the set
 
 Similar to the above, you will need to do the setup. You can then modify the version field to the version you wish to use. Once you have updated the package set, run the copy command and verify the package you have modified.
 
 Then you can make a pull request. Again, once Travis verifies your change, we will merge it into `master` and your change will be available in the next tag.
 
+### Add a package from Bower
+
+If you have Bower installed on your system, you can easily add a package from Bower to the package set. Follow the instructions above with the instructions written above. Then use the package name to add the package.
```

Comment:
>  Follow the instructions above with the instructions written above

That's a little repetitive little repetitive

Reply:
new keyboard woes...